### ISSUE TITLE
Can't access network drives

### ISSUE THREAD

**Issue by Steve01010** [files-community/Files#2275]
**Describe the bug**
The app crashes when trying to open network drives.

**To Reproduce**
1. Open a new tab
2. Change the path from 'Home' to a network drive path

**Expected behavior**
As in the default file explorer, the network drive should be opened and its contents should be visible.

**Desktop (please complete the following information):**
 - OS: Windows 10 1909
 - App version: 0.19.0.0

**Log file**
```
2020-11-08 16:39:51.6526|ERROR|Files.App|Invalid URI: The format of the URI could not be determined.|System.UriFormatException: Invalid URI: The format of the URI could not be determined.
   at SharedLibrary!<BaseAddress>+0xafcaa0
   at System.Uri..ctor(String) + 0x17
   at Files.Helpers.PathNormalization.NormalizePath(String) + 0xea
   at Files.UserControls.HorizontalMultitaskingControl.<SetSelectedTabInfo>d__10.MoveNext() + 0x65b
```

**Comment by gave92** [comment]
@Steve01010 thanks for this report! Could you post here the path you were trying to open?

**Comment by Steve01010** [comment]
for example, on windows file explorer, i see a network pane on the left sidebar... and i can click on any of the network drives

i can also type in the name of a network volume, for example, ``//HOME_NAS``, and it will open and display the contents

the same behavior crashes this application and throws the invalid path error, which i believe may be due to a lack of implementation of a feature to detect network drives and network drive paths

**Comment by gave92** [comment]
Thanks @Steve01010 a fix for this issue will be included in version 0.20.0
The fix will allow you to navigate to path like `\\DESKTOP-XXXX\Shared\SomePath`
